hubbot

```diff
@@ -437,21 +439,41 @@ def expire_images():
                     if 'tags' in conf:
                         tags = conf['tags']
                         for os_ in tags:
-                            keep_image (flavor, os_, tags[os_])
+                            keep_image(flavor, os_, tags[os_])
+                            keep_mock(flavor, os_, tags[os_])
                 except:
```

Comment:
I think this except in way too broad. Perhaps it should be moved up to the ```ast.literal_eval``` line?